parser-typechecker/src/Unison/Runtime/Machine.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE PatternSynonyms #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ViewPatterns #-}
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
```

Comment:
This file has a tone of uni-patterns that would be a pain to clean up;
If people feel super strongly about it we can see about going through them, but for now it's just as easy to disable this for only this file.

It's not something we check in our current 8.* GHC flags, so this isn't any worse off.

Reply:
Maybe we should have a `TODO`/`FIXME` about deleting this line and fixing the patterns?